features/nvstore/source/nvstore_int_flash_wrapper.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2018 ARM Limited. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+
+#ifndef __NVSTORE_INT_FLASH_WRAPPER_H
+#define __NVSTORE_INT_FLASH_WRAPPER_H
+
+#include <stdio.h>
+#include <stdint.h>
+
+/*
+ * This is a simple wrapper for the FlashIAP class, supplying basic flash functionality.
+ */
+
+#define NVSTORE_NUM_AREAS        2
```

Comment:
what is this number, it is not configurable, thus set to 2 (2 sectors). Is this constant value that is always set to two (only 2 areas are supported). Using 3 would fail or?  I could not locate the docs for this constant

Reply:
It is a part of NVStore logic. We simply have two areas, having an active a non-active area at all times. When the active area gets filled, we compact the values into the non-active area and switch activity. So this is a constant value used for clarity sake.

Reply:
Thanks, this I believe will be documented. Would it make sense to have this here in the comment 

I can think of:

```
// defines 2 areas - active and non-active, not configurable
#define NVSTORE_NUM_AREAS        2
```